src/test/java/seedu/address/model/person/assignment/initialise/AssignmentInitialiseTest.java

```diff
@@ -0,0 +1,14 @@
+package seedu.address.model.person.assignment.initialise;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+public class AssignmentInitialiseTest {
```

Comment:
would it be wise to add a test to assert that the Assignments cannot be reinitialised again? though i'm not sure how it would be done 😨